app/views/staff/events/show.html.haml

```diff
@@ -225,4 +225,4 @@
     -#           %p=event.speaker_notification_emails['waitlist'].truncate(75, separator: /\s/)
 
   -# %hr/
-  -# = render partial: 'event_teammates', locals: { event: event, event_teammates: event_teammates, rating_counts: rating_counts }
+  -# = render partial: 'teammates', locals: { event: event, teammates: teammates, rating_counts: rating_counts }
```

Comment:
You just mean all the commented out code, yes? I'm not sure who that belonged to originally